lms/static/lms/js/require-config.js

```diff
@@ -19,6 +19,12 @@
         };
         defineDependency("jQuery", "jquery");
         defineDependency("_", "underscore");
+        if (window._ && window._.str) {
```

Comment:
Right, I went down that road. But with the underscore.string upgrade, I can go back to the straightforward call to defineDependency because underscore.string won't be attached to "_" anymore. So it would just be wasted code.